mycroft/skills/npr_news/test/intent/sample2.intent.json

```diff
@@ -1,7 +1,7 @@
 {
-  "utterance": "play the news",
+  "utterance": "is there any news",
```

Comment:
If the npr test broke, it means we gonna have conflicts and trouble between media and npr skills. Unless we enforce the utterance somehow on one of the sides that'll still happen. Have we tried a real media skill using the keyword `play` and verify if it conflicts with npr?